theme_bewise/views/customizations.xml

```diff
@@ -317,6 +317,31 @@
     </xpath>
 </template>
 
+<!-- ======== FEATURES WALL ======== -->
+<template id="s_features_wall" inherit_id="website.s_features_wall">
+    <!-- Card 1 -->
+    <xpath expr="//div[hasclass('g-col-lg-3')]//h5" position="replace" mode="inner">
+        Innovative Academic Programs
+    </xpath>
+    <xpath expr="//div[hasclass('g-col-lg-3')]//p" position="replace" mode="inner">
+        Explore cutting-edge courses designed to prepare you for success in today’s evolving job market and academic fields.
+    </xpath>
+    <!-- Card 2 -->
+    <xpath expr="//div[hasclass('g-col-lg-3')][3]//h5" position="replace" mode="inner">
```

Comment:
Broken selector, small distraction I guess? 
```suggestion
    <xpath expr="//div[hasclass('g-col-lg-3')][2]//h5" position="replace" mode="inner">
```
**_OR_**
```suggestion
    <xpath expr="//div[hasclass('g-col-lg-3')][4]//h5" position="replace" mode="inner">
```

depending on which title you're trying to change 😉